src/api/rest/v1/connections/routes.ts

```diff
@@ -0,0 +1,12 @@
+import express from 'express'
+import Controller from './controller'
+
+const router = express.Router()
+
+router.get('/', Controller.connections)
+router.post('/sync', Controller.sync)
+router.post('/sync/:connectionId', Controller.syncConnection)
```

Comment:
To adhere with RESTFUL, it should be `/:connectionId/sync`

Reply:
Fixed